sopel/modules/reddit.py

```diff
@@ -249,9 +249,54 @@ def comment_info(bot, trigger, match):
     bot.say(message)
 
 
+def subreddit_sorting(bot, trigger, s, sorting):
+    if sorting == 'new':
+        submissions = list(s.new(limit=10))
+    elif sorting == 'top':
+        submissions = list(s.top(limit=10))
+    elif sorting == 'hot':
+        submissions = list(s.hot(limit=10))
+    elif sorting == 'controversial':
+        submissions = list(s.controversial(limit=10))
+    elif sorting == 'gilded':
+        submissions = list(s.gilded(limit=10))
+    elif sorting == 'rising':
+        submissions = list(s.rising(limit=10))
+    elif sorting == 'sticky':
+        try:
+            submissions = [s.sticky()]
+        except prawcore.exceptions.NotFound:
+            bot.say("r/" + s.display_name + " appears to not have a stickied post!")
```

Comment:
```suggestion
            bot.say("r/" + s.display_name + " appears not to have a stickied post!")
```

Split infinitives, oof.